---
title: "Chapters 9 & 11 Homework"
subtitle: "Ch. 9: 12, 13, 17, 18, 20, 21a, 24, 27, 35, 40; Ch.11: 3, 8, 9, 11, 13, 21"
author: "L E Delaney"
output:
  pdf_document:
      template: input/template.tex
      keep_tex: false 
      latex_engine: xelatex
  html_document:
    highlight: tango
    theme: cosmo
    toc: true
    toc_float: true
documentclass: "article"
fontsize: 11pt
geometry_options: margin = 1in
linestretch: 1
---
```{r setup, echo=FALSE, eval=TRUE, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, out.extra = "")
library(tidyverse)

myoutput <- "latex"
#myoutput <- "html"

#For alpha lists
alpha.list <- function(optnum = 3, opta, optb, optc, optd, output = myoutput){
  
  if(output == "html"){
    if(optnum == 3){
   a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>", 
       "\n","</ol>", sep = " ") %>%
     noquote()}
    else if(optnum == 2){
         a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n","</ol>", sep = " ") %>%
     noquote()
      
    }
    
     else if(optnum == 4){
        a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>",
       "\n", 
       "<li>",optd,"</li>",
       "\n","</ol>", sep = " ") %>%
             
     noquote()
      
    }
   
   return(a)
    
  }
  
  else if(output == "latex"){
    if(optnum == 3){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    else if(optnum == 2){
      b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()
      
    }
    
    else if(optnum == 4){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n", 
       "\\item[d.]{",optd,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    
    return(b)
  }
  
}

latex.only <- function(mytext, output = myoutput){
  
  if(output == "html"){
    return(NULL)
  }
  
  else if(output == "latex"){
    b <- paste0(mytext) %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(NULL)
    
  }
}

html.only <- function(mytext, output = myoutput){
  
  if(output == "latex"){
    return(NULL)
  }
  
  else if(output == "html"){
    b <- paste0(mytext) %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(NULL)
    
  }
}


bold.fix <- function(mytext, output = myoutput){
  
  if(output == "latex"){
    a <- paste0("\\textbf{", mytext, "}") %>%
      noquote()
    
    return(a)
  }
  
  else if(output == "html"){
    b <- paste0("<strong>", mytext, "</strong>") %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(mytext)
    
  }
}


```

:::: {.blackbox data-latex=""}

12. Use the codon dictionary in Figure 9-5 to complete the following table. Assume that reading is from left to right and that the columns represent transcriptional and translational alignments.

\

```{r, echo=FALSE, fig.align='center', out.width="75%"}

knitr::include_graphics(path = "input/q12.png")

```

\

Label the 5' and 3' ends of DNA and RNA, as well as the amino and carboxyl ends of the protein.

`r latex.only("\\vspace{12cm}")`

::::


:::: {.blackbox data-latex=""}

13. Consider the following segment of DNA:

:::: {.center data-latex=""}

5' GCTTCCCAA 3' 

3' CGAAGGGTT 5'

::::

Assume that the top strand is the template strand used by RNA polymerase.

`r alpha.list(optnum = 4, opta = "Draw the RNA transcribed.", optb = "Label its 5' and 3' ends.", optc = "Draw the corresponding amino acid chain.", optd = "Label its amino and carboxyl ends.")`


`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

17. Which anticodon would you predict for a tRNA species carrying isoleucine? Is there more than one possible answer? If so, state any alternative answers.


`r latex.only("\\vspace{21cm}")`

::::

:::: {.blackbox data-latex=""}

18. 
`r alpha.list(optnum = 2, opta = "In how many cases in the genetic code would you fail to know the amino acid specified by a codon if you knew only the first two nucleotides of the codon?", optb = "In how many cases would you fail to know the first two nucleotides of the codon if you knew which amino acid is specified by it?")`


`r latex.only("\\vspace{19cm}")`


::::

:::: {.blackbox data-latex=""}

20. If a polyribonucleotide contains equal amounts of randomly positioned adenine and uracil bases, what proportion of its triplets will encode (a) phenylalanine, (b) isoleucine, (c) leucine, and (d) tyrosine?


`r latex.only("\\vspace{21cm}")`

::::

:::: {.blackbox data-latex=""}

21a. You have synthesized three different messenger RNAs with bases incorporated in random sequence in the following ratios: **(a)** 1 U:5 C’s, (b) 1 A:1 C:4 U’s, (c) 1 A:1 C:1 G:1 U. In a protein-synthesizing system in vitro, indicate the identities and proportions of amino acids that will be incorporated into proteins when each of these mRNAs is tested. (Refer to Figure 9-5.)

```{r, echo=FALSE, fig.align='center', out.width="45%"}

knitr::include_graphics(path = "input/figure_09_05.jpg")

```

`r latex.only("\\vspace{13cm}")`

::::

:::: {.blackbox data-latex=""}

24. The enzyme tryptophan synthetase is produced in two sizes: large and small. Some mutants with no enzyme activity produced exactly the same size enzymes as the wild type. Other mutants with no activity produced just the large enzyme; still others, just the small enzyme.

`r alpha.list(optnum = 2, opta = "Explain the different types of mutants at the level of protein structure.", optb = "Why do you think there were no mutants that produced no enzyme?")`

`r latex.only("\\vspace{19cm}")`

::::


:::: {.blackbox data-latex=""}

27. A certain nonsense suppressor corrects a nongrowing mutant to a state that is near, but not exactly, wild type (it has abnormal growth). Suggest a possible reason why the reversion is not a full correction.


`r latex.only("\\vspace{21cm}")`

::::


:::: {.blackbox data-latex=""}

35. Explain why antibiotics that bind the large ribosomal subunit, such as erythromycin and Zithromax, do not harm us.

`r latex.only("\\vspace{21cm}")`

::::

:::: {.blackbox data-latex=""}

40. You are studying an _E. coli_ gene that specifies a protein. A part of its sequence is:

\

–Ala–Pro–Trp–Ser–Glu–Lys–Cys–His–

\

You recover a series of mutants for this gene that show no enzymatic activity. By isolating the mutant enzyme products, you find the following sequences:

\

Mutant 1: –Ala–Pro–Trp–Arg–Glu–Lys–Cys–His–

Mutant 2: –Ala–Pro–

Mutant 3: –Ala–Pro–Gly–Val–Lys–Asn–Cys–His–

Mutant 4: –Ala–Pro–Trp–Phe–Phe–Thr–Cys–His–

\

What is the molecular basis for each mutation? What is the DNA sequence that specifies this part of the protein?

`r latex.only("\\vspace{12cm}")`

::::

:::: {.blackbox data-latex=""}

3. Why do promoter mutations cluster at positions –10 and –35 as shown in Figure 11-11?

\

```{r, echo=FALSE, fig.align='center', out.width="75%"}

knitr::include_graphics(path = "input/figure_11_11.jpg")

```

`r latex.only("\\vspace{15cm}")`

::::

:::: {.blackbox data-latex=""}

8. Explain why $I^-$ alleles in the lac system are normally recessive to $I^+$ alleles and why $I^+$ alleles are
recessive to $I^S$ alleles.


`r latex.only("\\vspace{20cm}")`

::::

:::: {.blackbox data-latex=""}

9. What do we mean when we say that $O^C$ mutations in the _lac_ system are cis-acting?


`r latex.only("\\vspace{20cm}")`

::::

:::: {.blackbox data-latex=""}

11. The map of the lac operon is POZY.

The promoter (_P_) region is the start site of transcription through the binding of the RNA polymerase molecule before actual mRNA production. Mutationally altered promoters ($P^-$) apparently cannot bind the RNA polymerase molecule. Certain predictions can be made about the effect of P– mutations. Use your predictions and your knowledge of the lactose system to complete the following table. Insert a "+" where an enzyme is produced and a "–" where no enzyme is produced. The first one has been done as an example.

\

```{r, echo=FALSE, fig.align='center', out.width="85%"}

knitr::include_graphics(path = "input/q11.png")

```


`r latex.only("\\vspace{11cm}")`

::::

:::: {.blackbox data-latex=""}

13. Mutants that are $lacY^-$ retain the capacity to synthesize $\beta$-galactosidase. However, even though the _lacI_ gene is still intact, $\beta$-galactosidase can no longer be induced by adding lactose to the medium. Explain.


`r latex.only("\\vspace{17cm}")`

::::

:::: {.blackbox data-latex=""}

21. You are studying the properties of a new kind of regulatory mutation of the lactose operon. This mutation, called _S_, leads to the complete repression of the _lacZ_, _lacY_, and _lacA_ genes, regardless of whether inducer (lactose) is present. The results of studies of this mutation in partial diploids demonstrate that this mutation is completely dominant over wild type. When you treat bacteria of the _S_ mutant strain with a mutagen and select for mutant bacteria that can express the enzymes encoded by _lacZ_, _lacY_, and _lacA_ genes in the presence of lactose, some of the mutations map to the _lac_ operator region and others to the _lac_ repressor gene. On the basis of your knowledge of the lactose operon, provide a molecular genetic explanation for all these properties of the _S_ mutation. Include an explanation of the constitutive nature of the "reverse mutations."


`r latex.only("\\vspace{17cm}")`

::::

