---
title: "Chapters 6 & 7 Homework"
subtitle: "Ch. 6: 18, 21, 22, 60, 14, 15, 17, 44, 68; Ch.7: 6-10, 12-14, 15-17, 20, 25"
author: "L E Delaney"
output:
  pdf_document:
      template: input/template-landscape.tex
      keep_tex: false 
      latex_engine: xelatex
  html_document:
    highlight: tango
    theme: cosmo
    toc: true
    toc_float: true
documentclass: "article"
fontsize: 11pt
geometry_options: margin = 0.6in
linestretch: 1
---
```{r setup, echo=FALSE, eval=TRUE, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, out.extra = "")
library(tidyverse)

myoutput <- "latex"
#myoutput <- "html"

#For alpha lists
alpha.list <- function(optnum = 3, opta, optb, optc, optd, opte, output = myoutput){
  
  if(output == "html"){
    if(optnum == 3){
   a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>", 
       "\n","</ol>", sep = " ") %>%
     noquote()}
    else if(optnum == 2){
         a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n","</ol>", sep = " ") %>%
     noquote()
      
    }
    
     else if(optnum == 4){
        a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>",
       "\n", 
       "<li>",optd,"</li>",
       "\n","</ol>", sep = " ") %>%
             
     noquote()}
        
      else if(optnum == 5){
        a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>",
       "\n", 
       "<li>",optd,"</li>",
       "\n",
       "<li>",opte,"</li>",
       "\n","</ol>", sep = " ") %>%
             
     noquote()
      
    }
   
   return(a)
    
  }
  
  else if(output == "latex"){
    if(optnum == 3){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    else if(optnum == 2){
      b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()
      
    }
    
    else if(optnum == 4){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n", 
       "\\item[d.]{",optd,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    
  else if(optnum == 5){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n", 
       "\\item[d.]{",optd,"}", 
       "\n", 
       "\\item[e.]{",opte,"}",
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    
    return(b)
  }
  
}

latex.only <- function(mytext, output = myoutput){
  
  if(output == "html"){
    return(NULL)
  }
  
  else if(output == "latex"){
    b <- paste0(mytext) %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(NULL)
    
  }
}

html.only <- function(mytext, output = myoutput){
  
  if(output == "latex"){
    return(NULL)
  }
  
  else if(output == "html"){
    b <- paste0(mytext) %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(NULL)
    
  }
}


bold.fix <- function(mytext, output = myoutput){
  
  if(output == "latex"){
    a <- paste0("\\textbf{", mytext, "}") %>%
      noquote()
    
    return(a)
  }
  
  else if(output == "html"){
    b <- paste0("<strong>", mytext, "</strong>") %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(mytext)
    
  }
}


```

:::: {.blackbox data-latex=""}

18. If a man of blood-group AB marries a woman of blood-group A whose father was of blood-group O, to what different blood groups can this man and woman expect their children to belong?

`r latex.only("\\vspace{16cm}")`

::::


:::: {.blackbox data-latex=""}

21. In the multiple-allele series that determines coat color in rabbits, $c^+$ encodes agouti, $c^{ch}$ encodes chinchilla (a beige coat color), and $c^h$ encodes Himalayan. Dominance is in the order $c^+$ > $c^{ch}$ > $c^h$. In a cross of $c^+/c^{ch} \times c^{ch}/c^h$, what proportion of progeny will be chinchilla?


`r latex.only("\\vspace{16cm}")`

::::

:::: {.blackbox data-latex=""}

22. Black, sepia, cream, and albino are coat colors of guinea pigs. Individual animals (not necessarily from pure lines) showing these colors were intercrossed; the results are tabulated as follows, where the abbreviations A (albino), B (black), C (cream), and S (sepia) represent the following phenotypes:

```{r, echo=FALSE, fig.align='center', out.width="55%"}

knitr::include_graphics(path = "input/q22.png")

```

`r alpha.list(optnum = 2, opta = "Deduce the inheritance of these coat colors and use gene symbols of your own choosing. Show all parent and progeny genotypes.", optb = "If the black animals in crosses 7 and 8 are crossed, what progeny proportions can you predict by using your model?")`


`r latex.only("\\vspace{7cm}")`

::::

:::: {.blackbox data-latex=""}

60. Four homozygous recessive mutant lines of _Drosophila melanogaster_ (labeled 1 through 4) showed abnormal leg coordination, which made their walking highly erratic. These lines were intercrossed; the phenotypes of the F1 flies are shown in the following grid, in which "+" represents wild-type walking and "-" represents abnormal walking:

\

| Lines | 1 | 2 | 3 | 4 |
|-------|---|---|---|---|
|   1   | - | + | + | + |
|   2   | + | - | - | + |
|   3   | + | - | - | + |
|   4   | + | + | + | - |

\

`r alpha.list(optnum = 5, opta = "What type of test does this analysis represent?", optb = "How many different genes were mutated in creating these four lines?", optc = "Invent wild-type and mutant symbols and write out full genotypes for all four lines and for the F1 flies.", optd = "Do these data tell us which genes are linked? If not, how could linkage be tested?", opte = "Do these data tell us the total number of genes taking part in leg coordination in this animal?")`

`r latex.only("\\vspace{8cm}")`


::::

:::: {.blackbox data-latex=""}

14. In _Drosophila_, the autosomal recessive _bw_ causes a dark brown eye, and the unlinked autosomal recessive _st_ causes a bright scarlet eye. A homozygote for both genes has a white eye. Thus, we have the following correspondences between genotypes and phenotypes:

\

:::: {.center data-latex=""}

$st^+/ st^+$ ; $bw^+ / bw^+$ = red eye (wild type) 

$st^+/ st^+$ ; _bw_ / _bw_ = brown eye

_st_ / _st_ ; $bw^+ / bw^+$ = scarlet eye

_st_ / _st_ ; _bw_ / _bw_ = white eye

::::

\

Construct a hypothetical biosynthetic pathway showing how the gene products interact and why the different mutant combinations have different phenotypes.

`r latex.only("\\vspace{11cm}")`

::::

:::: {.blackbox data-latex=""}

15. Several mutants are isolated, all of which require compound G for growth. The compounds (A to E) in the biosynthetic pathway to G are known, but their order in the pathway is not known. Each compound is tested for its ability to support the growth of each mutant (1 to 5). In the following table, a plus sign indicates growth and a minus sign indicates no growth.

\

| Lines | A | B | C | D | E | G |
|-------|---|---|---|---|---|---|
|   1   | - | - | - | + | - | - |
|   2   | - | + | - | + | - | + |
|   3   | - | - | - | - | - | + |
|   4   | - | + | + | + | - | + |
|   5   | + | + | + | + | - | + |


`r alpha.list(optnum = 3, opta = "What is the order of compounds A to E in the pathway?", optb = "At which point in the pathway is each mutant blocked?", optc = "Would a heterokaryon composed of double mutants 1, 3 and 2, 4 grow on a minimal medium? Would 1, 3 and 3, 4? Would 1, 2 and 2, 4 and 1, 4?")`

`r latex.only("\\vspace{9cm}")`

::::

:::: {.blackbox data-latex=""}

17. In sweet peas, the synthesis of purple anthocyanin pigment in the petals is controlled by two genes, B and D. The pathway is:

\

White intermediate ---Gene B Enzyme---> blue intermediate ---Gene D Enzyme---> anthocyanin (purple)

\

`r alpha.list(optnum = 4, opta = "What color petals would you expect in a pure-breeding plant unable to catalyze the first reaction?", optb = "What color petals would you expect in a pure-breeding plant unable to catalyze the second reaction?", optc = "If the plants in parts a and b are crossed, what color petals will the F1 plants have?", optd = "What ratio of purple:blue:white plants would you expect in the F2?")`

`r latex.only("\\vspace{12cm}")`

::::

:::: {.blackbox data-latex=""}

44. The genotype _r/r_ ; _p/p_ gives fowl a single comb, _R/-_ ; _P/-_ gives a walnut comb, _r/r_ ; _P/-_ gives a pea comb, and _R/-_ ; _p/p_ gives a rose comb (see the illustrations). Assume independent assortment.

\

```{r, echo=FALSE, fig.align='center', out.width="55%"}

knitr::include_graphics(path = "input/q44.png")

```

`r alpha.list(optnum = 4, opta = "What comb types will appear in the F1 and in the F2 and in what proportions if single-combed birds are crossed with birds of a true-breeding walnut strain?", optb = "What are the genotypes of the parents in a walnut x rose mating from which the progeny are 3/8 rose, 3/8 walnut, 1/8 pea, and 1/8 single?", optc = "What are the genotypes of the parents in a walnut x rose mating from which all the progeny are walnut?", optd = "How many genotypes produce a walnut phenotype? Write them out.")`


`r latex.only("\\vspace{8cm}")`

::::

:::: {.blackbox data-latex=""}

68. Assume that two pigments, red and blue, mix to give the normal purple color of petunia petals. Separate biochemical pathways synthesize the two pigments, as shown in the top two rows of the accompanying diagram. "White" refers to compounds that are not pigments. (Total lack of pigment results in a white petal.) Red pigment forms from a yellow intermediate that is normally at a concentration too low to color petals.

\

```{r, echo=FALSE, fig.align='center', out.width="45%"}

knitr::include_graphics(path = "input/q68.png")

```


\

A third pathway, whose compounds do not contribute pigment to petals, normally does not affect the blue and red pathways, but, if one of its intermediates (white3) should build up in concentration, it can be converted into the yellow intermediate of the red pathway.

`r latex.only("\\vspace{0.5cm}")`

In the diagram, the letters A through E represent enzymes; their corresponding genes, all of which are unlinked, may be symbolized by the same letters.

`r latex.only("\\vspace{0.5cm}")`

Assume that wild-type alleles are dominant and encode enzyme function and that recessive alleles result in a lack of enzyme function. Deduce which combinations of true-breeding parental genotypes could be crossed to produce F2 progeny in the following ratios:

`r alpha.list(optnum = 4, opta = "9 purple:3 green:4 blue", optb = "9 purple:3 red:3 blue:1 white", optc = "13 purple:3 blue", optd = "9 purple:3 red:3 green:1 yellow")`

(**Note**: Blue mixed with yellow makes green; assume that no mutations are lethal.)


`r latex.only("\\vspace{4cm}")`

::::

:::: {.blackbox data-latex=""}

6. Describe the types of chemical bonds in the DNA double helix.

`r latex.only("\\vspace{7cm}")`

::::

:::: {.blackbox data-latex=""}

7. Explain what is meant by the terms _conservative_ and _semiconservative_ replication.

`r latex.only("\\vspace{7cm}")`

::::

:::: {.blackbox data-latex=""}

8. What is meant by a _primer_, and why are primers necessary for DNA replication?

`r latex.only("\\vspace{7cm}")`

::::

:::: {.blackbox data-latex=""}

9. What are helicases and topoisomerases?

`r latex.only("\\vspace{7cm}")`

::::

:::: {.blackbox data-latex=""}

10. Why is DNA synthesis continuous on one strand and discontinuous on the opposite strand?

`r latex.only("\\vspace{7cm}")`

::::

:::: {.blackbox data-latex=""}

12. If the helicases were missing during replication, what would happen to the replication process?

`r latex.only("\\vspace{7cm}")`

::::

:::: {.blackbox data-latex=""}

13. What would happen if, in the course of replication, the topoisomerases were unable to reattach the
DNA fragments of each strand after unwinding (relaxing) the DNA molecule?

`r latex.only("\\vspace{7cm}")`

::::

:::: {.blackbox data-latex=""}

14. Which of the following is _not_ a key property of hereditary material?

`r alpha.list(optnum = 4, opta = "It must be capable of being copied accurately.", optb = "It must encode the information necessary to form proteins and complex structures.", optc = "It must occasionally mutate.", optd = "It must be able to adapt itself to each of the body's tissues.")`

`r latex.only("\\vspace{6cm}")`

::::

:::: {.blackbox data-latex=""}

15. It is essential that RNA primers at the ends of Okazaki fragments be removed and replaced by DNA because otherwise which of the following events would result?

`r alpha.list(optnum = 4, opta = "The RNA would interfere with topoisomerase function.", optb = "The RNA would be more likely to contain errors because primase lacks a proofreading function.", optc = "The beta-clamp of the DNA pol II dimer would release the DNA and replication would stop.", optd = "The RNA primers would be likely to hydrogen bond to each other, forming complex structures that might interfere with the proper formation of the DNA helix.")`

`r latex.only("\\vspace{6cm}")`

::::

:::: {.blackbox data-latex=""}

16. Polymerases usually add only about 10 nucleotides to a DNA strand before dissociating. However, during replication, DNA pol III can add tens of thousands of nucleotides at a moving fork. How is this addition accomplished?

`r latex.only("\\vspace{6cm}")`

::::

:::: {.blackbox data-latex=""}

20. If the GC content of a DNA molecule is 48 percent, what are the percentages of the four bases (A, T, G, and C) in this molecule?

`r latex.only("\\vspace{5cm}")`

::::

:::: {.blackbox data-latex=""}

17. At each origin of replication, DNA synthesis proceeds bidirectionally from two replication forks. Which of the following would happen if a mutant arose having only one functional fork per replication bubble? (See diagram.)


```{r, echo=FALSE, fig.align='center', out.width="55%"}

knitr::include_graphics(path = "input/q17.png")

```

`r alpha.list(optnum = 4, opta = "There is no change at all in replication", optb = "Replication would take place only on one half of the chromosome.", optc = "Replication would be complete only on the leading strand.", optd = "Replication would take twice as long.")`

`r latex.only("\\vspace{4cm}")`

::::

:::: {.blackbox data-latex=""}

25. Consider the following segment of DNA, which is part of a much longer molecule constituting a chromosome:

\

:::: {.center data-latex=""}

5'....ATTCGTACGATCGACTGACTGACAGTC....3' 

3'....TAAGCATGCTAGCTGACTGACTGTCAG....5'

::::

\

If the DNA polymerase starts replicating this segment from the right,

`r alpha.list(optnum = 4, opta = "which will be the template for the leading strand?", optb = "draw the molecule when the DNA polymerase is halfway along this segment.", optc = "draw the two complete daughter molecules.", optd = "is your diagram in part b compatible with bidirectional replication from a single origin, the usual mode of replication?")`

`r latex.only("\\vspace{10cm}")`

::::


