---
title: "Chapters 10 & 16 Homework"
subtitle: "Ch. 10: 14, 15, 17, 18, 25, 32-37; Ch. 16: 8, 11, 13, 21, 25, 36"
author: "L E Delaney"
output:
  pdf_document:
      template: input/template.tex
      keep_tex: false 
      latex_engine: xelatex
  html_document:
    highlight: tango
    theme: cosmo
    toc: true
    toc_float: true
documentclass: "article"
fontsize: 11pt
geometry_options: margin = 1in
linestretch: 1
---
```{r setup, echo=FALSE, eval=TRUE, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, out.extra = "")
library(tidyverse)

myoutput <- "latex"
#myoutput <- "html"

#For alpha lists
alpha.list <- function(optnum = 3, opta, optb, optc, optd, output = myoutput){
  
  if(output == "html"){
    if(optnum == 3){
   a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>", 
       "\n","</ol>", sep = " ") %>%
     noquote()}
    else if(optnum == 2){
         a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n","</ol>", sep = " ") %>%
     noquote()
      
    }
    
     else if(optnum == 4){
        a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>",
       "\n", 
       "<li>",optd,"</li>",
       "\n","</ol>", sep = " ") %>%
             
     noquote()
      
    }
   
   return(a)
    
  }
  
  else if(output == "latex"){
    if(optnum == 3){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    else if(optnum == 2){
      b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()
      
    }
    
    else if(optnum == 4){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n", 
       "\\item[d.]{",optd,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    
    return(b)
  }
  
}

latex.only <- function(mytext, output = myoutput){
  
  if(output == "html"){
    return(NULL)
  }
  
  else if(output == "latex"){
    b <- paste0(mytext) %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(NULL)
    
  }
}

html.only <- function(mytext, output = myoutput){
  
  if(output == "latex"){
    return(NULL)
  }
  
  else if(output == "html"){
    b <- paste0(mytext) %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(NULL)
    
  }
}


bold.fix <- function(mytext, output = myoutput){
  
  if(output == "latex"){
    a <- paste0("\\textbf{", mytext, "}") %>%
      noquote()
    
    return(a)
  }
  
  else if(output == "html"){
    b <- paste0("<strong>", mytext, "</strong>") %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(mytext)
    
  }
}


```

:::: {.blackbox data-latex=""}

14. Why is ligase needed to make recombinant DNA? What would be the immediate consequence in the cloning process if someone forgot to add it?

`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

15. In the PCR process, if we assume that each cycle takes 5 minutes, how many-fold amplification would be accomplished in 1 hour?

`r latex.only("\\vspace{20cm}")`

::::

:::: {.blackbox data-latex=""}

17. You obtain the DNA sequence of a mutant of a 2-kb gene in which you are interested and it shows base differences at three positions, all in different codons. One is a silent change, but the other two are missense changes (they encode new amino acids). How would you demonstrate that these changes are real mutations and not sequencing errors? (Assume that sequencing is about 99.9 percent accurate.)

`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

18. In a T-DNA transformation of a plant with a transgene from a fungus (not found in plants), the presumptive transgenic plant does not express the expected phenotype of the transgene. How would you demonstrate that the transgene is in fact present? How would you demonstrate that the transgene was expressed?


`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

25. In yeast, you have sequenced a piece of wild-type DNA and it clearly contains a gene, but you do not know what gene it is. Therefore, to investigate further, you would like to find out its mutant phenotype. How would you use the cloned wild-type gene to do so? Show your experimental steps clearly.

`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

32. A cloned fragment of DNA was sequenced by using the dideoxy chain-termination method. A part of the autoradiogram of the sequencing gel is represented here.

\

```{r, echo=FALSE, fig.align='center', out.width="45%"}

knitr::include_graphics(path = "input/problem_10_32.jpg")

```

`r alpha.list(optnum = 3, opta = "Deduce the nucleotide sequence of the DNA nucleotide chain synthesized from the primer. Label the 5' and 3' ends.", optb = "Deduce the nucleotide sequence of the DNA nucleotide chain used as the template strand. Label the 5' and 3' ends.", optc = "Write out the nucleotide sequence of the DNA double helix (label the 5' and 3' ends).")`

`r latex.only("\\vspace{8cm}")`

::::

:::: {.blackbox data-latex=""}

33. The cDNA clone for the human gene encoding tyrosinase was radioactively labeled and used in a Southern analysis of _EcoRI_-digested genomic DNA of wild-type mice. Three mouse fragments were found to be radioactive (were bound by the probe). When albino mice were used in this Southern analysis, no genomic fragments bound to the probe. Explain these results in relation to the nature of the wild-type and mutant mouse alleles.


`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

34. Transgenic tobacco plants were obtained in which the vector Ti plasmid was designed to insert the gene of interest plus an adjacent kanamycin-resistance gene. The inheritance of chromosomal insertion was followed by testing progeny for kanamycin resistance. Two plants typified the results obtained generally. When plant 1 was backcrossed with wild-type tobacco, 50 percent of the progeny were kanamycin resistant and 50 percent were sensitive. When plant 2 was backcrossed with the wild type, 75 percent of the progeny were kanamycin resistant and 25 percent were sensitive. What must have been the difference between the two transgenic plants? What would you predict about the situation regarding the gene of interest?


`r latex.only("\\vspace{17cm}")`

::::

:::: {.blackbox data-latex=""}

35. A cystic-fibrosis mutation in a certain pedigree is due to a single nucleotide-pair change. This change destroys an _EcoRI_ restriction site normally found in this position. How would you use this information in counseling members of this family about their likelihood of being carriers? State the precise experiments needed. Assume that you find that a woman in this family is a carrier, and it transpires that she is married to an unrelated man who also is a heterozygote for cystic fibrosis, but, in his case, it is a different mutation in the same gene. How would you counsel this couple about the risks of a child’s having cystic fibrosis?

`r latex.only("\\vspace{18cm}")`

::::

:::: {.blackbox data-latex=""}

36. Bacterial glucuronidase converts a colorless substance called X-Gluc into a bright blue indigo pigment. The gene for glucuronidase also works in plants if given a plant promoter region. How would you use this gene as a reporter gene to find the tissues in which a plant gene that you have just cloned is normally active? (Assume that X-Gluc is easily taken up by the plant tissues.)


`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

37. The plant _Arabidopsis thaliana_ was transformed by using the Ti plasmid into which a kanamycin-resistance gene had been inserted in the T-DNA region. Two kanamycin-resistant colonies (A and B) were selected, and plants were regenerated from them. The plants were allowed to self-pollinate, and the results were as follows:

\

Plant A selfed → 3/4 progeny resistant to kanamycin; 1/4 progeny sensitive to kanamycin

Plant B selfed → 5/16 progeny resistant to kanamycin; 1/16 progeny sensitive to kanamycin

\

a. Draw the relevant plant chromosomes in both plants.

b. Explain the two different ratios.

\


`r latex.only("\\vspace{16cm}")`

::::


:::: {.blackbox data-latex=""}

8. Consider the following wild-type and mutant sequences:


:::: {.center data-latex=""}

Wild-type ....CTTGCAAGCGAATC.... 

Mutant ....CTTGCTAGCGAATC....

::::

The substitution shown seems to have created a stop codon. What further information do you need
to be confident that it has done so?

`r latex.only("\\vspace{17cm}")`

::::


:::: {.blackbox data-latex=""}

11. By base-pair substitution, what are all the synonymous changes that can be made starting with the codon CGG?


`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

13. 
`r alpha.list(optnum = 2, opta = "Acridine orange is an effective mutagen for producing null alleles by mutation. Why does it produce null alleles?", optb = "A certain acridine-like compound generates only single insertions. A mutation induced with this compound is treated with the same compound, and some revertants are produced. How is this outcome possible?")`

`r latex.only("\\vspace{19cm}")`

::::

:::: {.blackbox data-latex=""}

21. Differentiate between the elements of the following pairs: 

a. Transitions and transversions

b. Synonymous and neutral mutations

c. Missense and nonsense mutations

d. Frameshift and nonsense mutations


`r latex.only("\\vspace{19cm}")`


::::

:::: {.blackbox data-latex=""}

25. A certain compound that is an analog of the base cytosine can become incorporated into DNA. It normally hydrogen bonds just as cytosine does, but it quite often isomerizes to a form that hydrogen bonds as thymine does. Do you expect this compound to be mutagenic, and, if so, what types of changes might it induce at the DNA level?


`r latex.only("\\vspace{20cm}")`

::::

:::: {.blackbox data-latex=""}

36. You are working with a newly discovered mutagen, and you wish to determine the base change that it introduces into DNA. Thus far, you have determined that the mutagen chemically alters a single base in such a way that its base-pairing properties are altered permanently. To determine the specificity of the alteration, you examine the amino acid changes that take place after mutagenesis. A sample of what you find is shown here:


:::: {.center data-latex=""}

Original: Gln–His–Ile–Glu–Lys

Mutant: Gln–His–Met–Glu–Lys

Original: Ala–Val–Asn–Arg

Mutant: Ala–Val–Ser–Arg

Original: Arg–Ser–Leu

Mutant:Arg–Ser–Leu–Trp–Lys–Thr–Phe 

::::

\

What is the base-change specificity of the mutagen?

`r latex.only("\\vspace{15cm}")`

::::

