---
title: "Chapters 10 & 16 Homework"
subtitle: "Ch. 10: 14, 15, 17, 18, 25, 32-37; Ch. 16: 8, 11, 13, 21, 25, 36"
author: "L E Delaney"
output:
  pdf_document:
      template: input/template.tex
      keep_tex: false 
      latex_engine: xelatex
  html_document:
    highlight: tango
    theme: cosmo
    toc: true
    toc_float: true
documentclass: "article"
fontsize: 11pt
geometry_options: margin = 1in
linestretch: 1
---
```{r setup, echo=FALSE, eval=TRUE, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, out.extra = "")
library(tidyverse)

myoutput <- "latex"
#myoutput <- "html"

#For alpha lists
alpha.list <- function(optnum = 3, opta, optb, optc, optd, output = myoutput){
  
  if(output == "html"){
    if(optnum == 3){
   a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>", 
       "\n","</ol>", sep = " ") %>%
     noquote()}
    else if(optnum == 2){
         a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n","</ol>", sep = " ") %>%
     noquote()
      
    }
    
     else if(optnum == 4){
        a <- paste("<ol type='a'>", 
       "\n", 
       "<li>",opta,"</li>", 
       "\n", 
       "<li>",optb,"</li>", 
       "\n", 
       "<li>",optc,"</li>",
       "\n", 
       "<li>",optd,"</li>",
       "\n","</ol>", sep = " ") %>%
             
     noquote()
      
    }
   
   return(a)
    
  }
  
  else if(output == "latex"){
    if(optnum == 3){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    else if(optnum == 2){
      b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()
      
    }
    
    else if(optnum == 4){
    b <- paste("\\begin{enumerate}", 
       "\n", 
       "\\item[a.]{",opta,"}", 
       "\n", 
       "\\item[b.]{",optb,"}", 
       "\n", 
       "\\item[c.]{",optc,"}", 
       "\n", 
       "\\item[d.]{",optd,"}", 
       "\n","\\end{enumerate}", sep = " ") %>%
      noquote()}
    
    return(b)
  }
  
}

latex.only <- function(mytext, output = myoutput){
  
  if(output == "html"){
    return(NULL)
  }
  
  else if(output == "latex"){
    b <- paste0(mytext) %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(NULL)
    
  }
}

html.only <- function(mytext, output = myoutput){
  
  if(output == "latex"){
    return(NULL)
  }
  
  else if(output == "html"){
    b <- paste0(mytext) %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(NULL)
    
  }
}


bold.fix <- function(mytext, output = myoutput){
  
  if(output == "latex"){
    a <- paste0("\\textbf{", mytext, "}") %>%
      noquote()
    
    return(a)
  }
  
  else if(output == "html"){
    b <- paste0("<strong>", mytext, "</strong>") %>%
      noquote()
    return(b)
  }
  
  else if(output == "word"){
    return(mytext)
    
  }
}


```

:::: {.blackbox data-latex=""}

14. Why is ligase needed to make recombinant DNA? What would be the immediate consequence in the cloning process if someone forgot to add it?


\

Ligase is an essential enzyme within all cells that seals breaks in the phosphate–sugar backbone of DNA. During DNA replication, it joins Okazaki fragments to create a continuous strand, and in cloning, it is used to join the various DNA fragments with the vector. If it were not added, the vector and cloned DNA would simply fall apart.

\

::::

:::: {.blackbox data-latex=""}

15. In the PCR process, if we assume that each cycle takes 5 minutes, how many-fold amplification would be accomplished in 1 hour?

\

Each cycle takes 5 minutes and doubles the DNA. In 1 hour, there would be 12 cycles, so the DNA would be amplified $2^{12} = 4096$-fold amplification.

\

::::

:::: {.blackbox data-latex=""}

17. You obtain the DNA sequence of a mutant of a 2-kb gene in which you are interested and it shows base differences at three positions, all in different codons. One is a silent change, but the other two are missense changes (they encode new amino acids). How would you demonstrate that these changes are real mutations and not sequencing errors? (Assume that sequencing is about 99.9 percent accurate.)

\

Resequencing the relevant gene should be done, as this will tell you if the original sequence was correct. You can then use the mutant sequence in a gene replacement experiment (depending on the organism) to see if the mutant phenotype is actually the result of the sequence variation you detected.

\

::::

:::: {.blackbox data-latex=""}

18. In a T-DNA transformation of a plant with a transgene from a fungus (not found in plants), the presumptive transgenic plant does not express the expected phenotype of the transgene. How would you demonstrate that the transgene is in fact present? How would you demonstrate that the transgene was expressed?

\

You could isolate DNA from the suspected transgenic plant and probe for the presence of the transgene by Southern hybridization.

\

::::

:::: {.blackbox data-latex=""}

25. In yeast, you have sequenced a piece of wild-type DNA and it clearly contains a gene, but you do not know what gene it is. Therefore, to investigate further, you would like to find out its mutant phenotype. How would you use the cloned wild-type gene to do so? Show your experimental steps clearly.

\

The typical procedure is to "knock out" the gene in question and then see if there is any observable phenotype. One methodology for doing this is described in the textbook. A recombinant vector carrying a selectable gene within the gene of interest is used to transform yeast cells. Grown under appropriate conditions, yeast that have incorporated the marker gene will be selected. Many of these will have the gene of interest disrupted by the selectable gene. The phenotype of these cells would then be assessed to determine gene function.

\

::::

:::: {.blackbox data-latex=""}

32. A cloned fragment of DNA was sequenced by using the dideoxy chain-termination method. A part of the autoradiogram of the sequencing gel is represented here.

\

```{r, echo=FALSE, fig.align='center', out.width="45%"}

knitr::include_graphics(path = "input/problem_10_32.jpg")

```

\

a. The gel can be read from the bottom to the top in a 5′-to-3′ direction. The sequence is:

5′ TTCGAAAGGTGACCCCTGGACCTTTAGA 3′

\

b. By complementarity, the template was

3′ AAGCTTTCCACTGGGGACCTGGAAATCT 5′

\

c. The double helix is

5′ TTCGAAAGGTGACCCCTGGACCTTTAGA 3′ 

3′ AAGCTTTCCACTGGGGACCTGGAAATCT 5′

\

::::

:::: {.blackbox data-latex=""}

33. The cDNA clone for the human gene encoding tyrosinase was radioactively labeled and used in a Southern analysis of _EcoRI_-digested genomic DNA of wild-type mice. Three mouse fragments were found to be radioactive (were bound by the probe). When albino mice were used in this Southern analysis, no genomic fragments bound to the probe. Explain these results in relation to the nature of the wild-type and mutant mouse alleles.

\

The region of DNA that encodes tyrosinase in "normal" mouse genomic DNA contains two _EcoRI_ sites. Thus, after _EcoRI_ digestion, three different-sized fragments hybridize to the cDNA clone. When genomic DNA from certain albino mice is subjected to similar analysis, there are no DNA fragments that contain complementary sequences to the same cDNA. This indicates that these mice lack the ability to produce tyrosinase because the DNA that encodes the enzyme must be deleted.

\

::::

:::: {.blackbox data-latex=""}

34. Transgenic tobacco plants were obtained in which the vector Ti plasmid was designed to insert the gene of interest plus an adjacent kanamycin-resistance gene. The inheritance of chromosomal insertion was followed by testing progeny for kanamycin resistance. Two plants typified the results obtained generally. When plant 1 was backcrossed with wild-type tobacco, 50 percent of the progeny were kanamycin resistant and 50 percent were sensitive. When plant 2 was backcrossed with the wild type, 75 percent of the progeny were kanamycin resistant and 25 percent were sensitive. What must have been the difference between the two transgenic plants? What would you predict about the situation regarding the gene of interest?

\

Plant 1 shows the typical inheritance for a dominant gene that is heterozygous. Assuming kanamycin resistance is dominant to kanamycin sensitivity, the cross can be outlined as follows:

\

$kan^R/kan^S \times kan^S/kan^S$


$\frac{1}{2} kan^R/kan^S$ 

$\frac{1}{2} kan^S/kan^S$

This would suggest that the gene of interest would be inserted once into the genome.

\

Plant 2 shows a 3:1 ratio in the progeny of the backcross. This suggests that there have been two
unlinked insertions of the kanR gene and presumably the gene of interest as well.

\

$kan^R1/kan^S1 ; kan^R2/kan^S2 \times kan^S1/kan^S1 ; kan^S2/kan^S2$

\

$\frac{1}{4} kan^{R1}/kan^{S1} ; kan^{R2}/kan^{S2}$

$\frac{1}{4} kan^{R1}/kan^{S1} ; kan^{S2}/kan^{S2}$

$\frac{1}{4} kan^{S1}/kan^{S1} ; kan^{R2}/kan^{S2}$

$\frac{1}{4} kan^{S1}/kan^{S1} ; kan^{R2}/kan^{S2}$

\


::::

:::: {.blackbox data-latex=""}

35. A cystic-fibrosis mutation in a certain pedigree is due to a single nucleotide-pair change. This change destroys an _EcoRI_ restriction site normally found in this position. How would you use this information in counseling members of this family about their likelihood of being carriers? State the precise experiments needed. Assume that you find that a woman in this family is a carrier, and it transpires that she is married to an unrelated man who also is a heterozygote for cystic fibrosis, but, in his case, it is a different mutation in the same gene. How would you counsel this couple about the risks of a child’s having cystic fibrosis?

\

Assuming that the DNA from this region is cloned, it could be used as a probe to detect this RFLP on Southern blots. DNA from individuals within this pedigree would be isolated (typically from blood samples containing white blood cells) and restricted with _EcoRI_, and Southern blots would be performed. Individuals with this mutant CF allele would have one band that would be larger (owing to the missing _EcoRI_ site) when compared with wild type. Individuals that inherited this larger _EcoRI_ fragment would, at minimum, be carriers for cystic fibrosis. In the specific case discussed in this problem, a woman that is heterozygous for this specific allele marries a man that is heterozygous for a different mutated CF allele. Just knowing that both are heterozygous, it is possible to predict that there is a 25-percent chance of their child’s having CF. However, because the mother’s allele is detectable on a Southern blot, it would be possible to test whether the fetus inherited this allele. DNA from the fetus (through either CVS or amniocentesis) could be isolated and tested for this specific EcoRI fragment. If the fetus did not inherit this allele, there would be a 0 percent chance of its having CF. On the other hand, if the fetus inherited this allele, there would be a 50-percent chance the child will have CF.

\

::::

:::: {.blackbox data-latex=""}

36. Bacterial glucuronidase converts a colorless substance called X-Gluc into a bright blue indigo pigment. The gene for glucuronidase also works in plants if given a plant promoter region. How would you use this gene as a reporter gene to find the tissues in which a plant gene that you have just cloned is normally active? (Assume that X-Gluc is easily taken up by the plant tissues.)


\

The promoter and control regions of the plant gene of interest must be cloned and joined in the correct orientation with the glucuronidase gene. This places the reporter gene under the same transcriptional control as the gene of interest. The textbook discusses the methodology used to create transgenic plants. Transform plant cells with the reporter gene construct, and as discussed in the textbook, grow into transgenic plants. The glucuronidase gene will now be expressed in the same developmental pattern as the gene of interest and its expression can easily be monitored by bathing the plant in an X-Gluc solution and assaying for the blue reaction product.

\

::::

:::: {.blackbox data-latex=""}

37. The plant _Arabidopsis thaliana_ was transformed by using the Ti plasmid into which a kanamycin-resistance gene had been inserted in the T-DNA region. Two kanamycin-resistant colonies (A and B) were selected, and plants were regenerated from them. The plants were allowed to self-pollinate, and the results were as follows:

\

Plant A selfed → 3/4 progeny resistant to kanamycin; 1/4 progeny sensitive to kanamycin

Plant B selfed → 5/16 progeny resistant to kanamycin; 1/16 progeny sensitive to kanamycin

\

a. Draw the relevant plant chromosomes in both plants.

b. Explain the two different ratios.

\


**a. and b.** During Ti plasmid transformation, the kanamycin gene will insert randomly into the plant
chromosomes. Colony A, when selfed, has 3/4 kanamycin-resistant progeny, and colony B, when selfed, has 15/16 kanamycin-resistant progeny. This suggests that there was a single insertion into one chromosome in colony A and two independent insertions on separate chromosomes in colony B. This can be schematically represented by showing a single insertion within one of the pair of chromosome "A" for colony A and two independent insertions into one of each of the pairs of chromosomes "B" and "C" for colony B.

```{r, echo=FALSE, fig.align='center', out.width="45%"}

knitr::include_graphics(path = "input/problem37.png")
knitr::include_graphics(path = "input/question37-2.png")

```

\

::::


:::: {.blackbox data-latex=""}

8. Consider the following wild-type and mutant sequences:


:::: {.center data-latex=""}

Wild-type ....CTTGCAAGCGAATC.... 

Mutant ....CTTGCTAGCGAATC....

::::

The substitution shown seems to have created a stop codon. What further information do you need
to be confident that it has done so?

\

You need to know the reading frame of the possible message.

::::


:::: {.blackbox data-latex=""}

11. By base-pair substitution, what are all the synonymous changes that can be made starting with the codon CGG?


\

Assuming single base-pair substitutions, then CGG can be changed to CGU, CGA, CGC, or AGG and still would code for arginine.

\

::::

:::: {.blackbox data-latex=""}

13. 
`r alpha.list(optnum = 2, opta = "Acridine orange is an effective mutagen for producing null alleles by mutation. Why does it produce null alleles?", optb = "A certain acridine-like compound generates only single insertions. A mutation induced with this compound is treated with the same compound, and some revertants are produced. How is this outcome possible?")`

\

a. Acridine orange causes frameshift mutations and frameshift mutations often result in null alleles.

b. A+1 frameshift mutation can be reverted by two further single insertions so that the reading
frame is re-established.

\

::::

:::: {.blackbox data-latex=""}

21. Differentiate between the elements of the following pairs: 

a. Transitions and transversions

b. Synonymous and neutral mutations

c. Missense and nonsense mutations

d. Frameshift and nonsense mutations


\

a. A transition mutation is the substitution of a purine for a purine or the substitution of a pyrimidine for a pyrimidine. A transversion mutation is the substitution of a purine for a pyrimidine, or vice versa.

b. Both are base-pair substitutions. A synonymous mutation is one that does not alter the amino acid sequence of the protein product from the gene because the new codon codes for the same amino acid as did the nonmutant codon. A neutral mutation results in a different amino acid that is functionally equivalent, and the mutation therefore has no known adaptive significance.

c. A missense mutation results in a different amino acid in the protein product of the gene. A nonsense mutation causes premature termination of translation, resulting in a shortened protein. d. Frameshift mutations arise from addition or deletion of one or more bases in other than multiples of three, thus altering the reading frame for translation. Therefore, the amino acid sequence from the site of the mutation to the end of the protein product of the gene will be altered. Frameshift mutations can and often do result in premature stop codons in the new reading frame, leading to shortened protein products. A nonsense mutation causes premature termination of translation in the original reading frame, resulting in a shortened protein.

\

::::

:::: {.blackbox data-latex=""}

25. A certain compound that is an analog of the base cytosine can become incorporated into DNA. It normally hydrogen bonds just as cytosine does, but it quite often isomerizes to a form that hydrogen bonds as thymine does. Do you expect this compound to be mutagenic, and, if so, what types of changes might it induce at the DNA level?


\

Yes. It will cause CG-to-TA transitions.

::::

:::: {.blackbox data-latex=""}

36. You are working with a newly discovered mutagen, and you wish to determine the base change that it introduces into DNA. Thus far, you have determined that the mutagen chemically alters a single base in such a way that its base-pairing properties are altered permanently. To determine the specificity of the alteration, you examine the amino acid changes that take place after mutagenesis. A sample of what you find is shown here:


:::: {.center data-latex=""}

Original: Gln–His–Ile–Glu–Lys

Mutant: Gln–His–Met–Glu–Lys

Original: Ala–Val–Asn–Arg

Mutant: Ala–Val–Ser–Arg

Original: Arg–Ser–Leu

Mutant:Arg–Ser–Leu–Trp–Lys–Thr–Phe 

::::

\

What is the base-change specificity of the mutagen?

\

Compare the original amino acid sequences with the mutant ones and list the changes.

original: ile; mutant: met 

original: asn; mutant: ser 

original: stop; mutant: trp

\

Now compare the codons that must have been altered by this mutagen.

original: ile AUA;

original: asn AAC or AAU; 

original: stop UAG or UGA;

mutant: met AUG 

mutant: ser AGC or AGU mutant: trp UGG

\

All these mutations can be the result of T to C or A to G transitions in the DNA. The result would be an A to G change in the mRNA that explains all three codon changes. This mutagen, then, might work by altering the base-pairing specificity of T so that it now base pairs with G. Or, the mutagen could alter the pairing specificity of A so that it now pairs with C, which would have the same effect.

\

::::

